semgrep-monitor

```diff
@@ -73,13 +130,57 @@ def main():
         "username": "Semgrep",
     }
     try:
-        urllib.request.urlopen(
-            args.slack_url,
-            urllib.parse.urlencode({"payload": json.dumps(payload)}).encode(),
+        response = requests.post(args.slack_url, data={"payload": json.dumps(payload)},)
+        response.raise_for_status()
+    except requests.RequestException:
+        print(f"Slack returned this error: {response.text}", file=sys.stderr)
+
+
+def report_finish_to_semgrep_app():
+    if not args.semgrep_app_token:
+        return
+
+    # send findings
+    results = json.load(args.results_file)
+    payload = [
```

Comment:
We may want to batch this in groups of 10000 or so to keep the payload size below 10 MB.

Reply:
(See `bento.util.batched`)